docs/rendering-text.md

```diff
@@ -0,0 +1,19 @@
+# Building documentation
+
+Jupyter Books natively render **.rst, .md, and .ipynb** files natively from a single GitHub repo in a beautiful way.
```

Comment:
Jupyter Books natively render **.rst, .md, and .ipynb** files from a single GitHub repo in a beautiful way.

-removed "naively" twice